server/settings/publish_plugins.py

```diff
@@ -4,19 +4,140 @@
     enabled: bool = SettingsField(False, title="Enabled")
 
 
-class CollectDeadlinePoolsModel(BaseSettingsModel):
-    """Settings Deadline default pools."""
+def extract_jobinfo_overrides_enum():
+    """Enum of fields that could be overridden by artist in Publisher UI"""
+    return [
+        {"label": "Department", "value": "department"},
+        {"label": "Delay job (timecode dd:hh:mm:ss)", "value": "job_delay"},
+        {"label": "Frames per Task", "value": "chunk_size"},
+        {"label": "Group", "value": "group"},
+        {"label": "Priority", "value": "priority"},
+        {"label": "Limit groups", "value": "limit_groups"},
+        {"label": "Primary pool", "value": "primary_pool"},
+        {"label": "Secondary pool", "value": "secondary_pool"},
+        {"label": "Machine List", "value": "machine_list"},
+        {"label": "Machine List is a Deny", "value": "machine_list_deny"},
+    ]
 
-    primary_pool: str = SettingsField(title="Primary Pool")
 
-    secondary_pool: str = SettingsField(title="Secondary Pool")
+class CollectJobInfoItem(BaseSettingsModel):
+    _layout = "expanded"
+    host_names: list[str] = SettingsField(
+        default_factory=list,
+        title="Host names"
+    )
+    task_types: list[str] = SettingsField(
+        default_factory=list,
+        title="Task types",
+        enum_resolver=task_types_enum
+    )
+    task_names: list[str] = SettingsField(
+        default_factory=list,
+        title="Task names"
+    )
+
+    #########################################
+
+    chunk_size: int = SettingsField(999, title="Frames per Task")
+    priority: int = SettingsField(50, title="Priority")
+    group: str = SettingsField("", title="Group")
+    limit_groups: list[str] = SettingsField(
+        default_factory=list,
+        title="Limit Groups"
+    )
+    primary_pool: str = SettingsField("", title="Primary Pool")
+    secondary_pool: str = SettingsField("", title="Secondary Pool")
+    machine_limit: int = SettingsField(
+        0,
+        title="Machine Limit",
+        description=(
+            "Specifies the maximum number of machines this job can be"
+            " rendered on at the same time (default = 0, which means"
+            " unlimited)."
+        )
+    )
+    machine_list: list[str] = SettingsField(
+        default_factory=list,
+        title="Machine List",
+        description=(
+            "List of workers where submission can/cannot run "
+            "based on Machine Allow/Deny toggle."
+        )
+    )
+    machine_list_deny: bool = SettingsField(
+        False, title="Machine List is a Deny",
+        description=(
+            "Explicitly DENY list of machines to render. Without it "
+            "it will ONLY ALLOW machines from list."
+        )
+    )
+    concurrent_tasks: int = SettingsField(
+        1, title="Number of concurrent tasks")
+    department: str = SettingsField("", title="Department")
+    use_gpu: bool = SettingsField("", title="Use GPU")
```

Comment:
Hey, 
Is this attribute exclusive to some plugings?
(because I'm mostly on Houdini, I can't recall this attribute)
I checked the code changes in this PR, [Manual Job Submission](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/manual-submission.html) and compared jop properties for Houdini and Nuke. 

and I think this is attribute is exclusive to some plugins like Nuke and fusion. 

Anyway, I'm just asking so that I can mentioned that in documentation.
![image](https://github.com/user-attachments/assets/ccb63e31-c691-4c80-9e3f-46c85ff9dbf5)

Reply:
Yeah, you are right, will be removing.